package.json

```diff
@@ -58,7 +58,8 @@
     "configPath": "tests/dummy/config",
     "fastbootDependencies": [
       "node-fetch",
-      "abortcontroller-polyfill"
+      "abortcontroller-polyfill",
```

Comment:
Should be able to delete this line

Reply:
You're right. Changed, tested, and updated the PR. Thanks!